lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -439,13 +440,25 @@ class DefaultAssetPickerBuilderDelegate
           allowSpecialItemWhenEmpty: allowSpecialItemWhenEmpty,
         );
 
+  /// Thumbnail size in the grid.
+  /// 预览时网络的缩略图大小
+  ///
+  /// This only works on images and videos since other types does not have to
+  /// request for the thumbnail data. The preview can speed up by reducing it.
+  /// 该参数仅生效于图片和视频类型的资源，因为其他资源不需要请求缩略图数据。
+  /// 预览图片的速度可以通过适当降低它的数值来提升。
+  ///
+  /// This cannot be `null` or a large value since you shouldn't use the
+  /// original data for the grid.
+  /// 该值不能为空或者非常大，因为在网格中使用原数据不是一个好的决定。
+  final int gridThumbSize;
+
   /// Preview thumbnail size in the viewer.
   /// 预览时图片的缩略图大小
   ///
-  /// This only works on images since other types does not have request
-  /// for thumb data. The speed of preview can be raised by reducing it.
-  ///
-  /// 该参数仅生效于图片类型的资源，因为其他资源不需要请求缩略图数据。
+  /// This only works on images and videos since other types does not have to
+  /// request for the thumbnail data. The preview can speed up by reducing it.
+  /// 该参数仅生效于图片和视频类型的资源，因为其他资源不需要请求缩略图数据。
   /// 预览图片的速度可以通过适当降低它的数值来提升。
   ///
   /// Default is `null`, which will request the origin data.
```

Comment:
`/// Default is null, which will request the origin data.`

I think this is incorrect. If you provide `null`, you will not get the original but the default preview size.
That is because you are providing `isOriginal: false` to `AssetEntityImageProvider`.